editor.planx.uk/src/routes/flowSettings.tsx

```diff
@@ -58,32 +84,20 @@ const flowSettingsRoutes = compose(
         const settings: FlowSettings = data.flows[0].settings;
         useStore.getState().setFlowSettings(settings);
 
+        function getTabs() {
+          const isUsingFeatureFlag = hasFeatureFlag(
+            "APPLICATION_SUBMISSION_VIEW",
+          );
+          return isUsingFeatureFlag
+            ? [...standardTabs, submissionsTab]
+            : standardTabs;
+        }
```

Comment:
Should the visibility of the `submissionsTab` also be scoped by `canUserEditTeam()`?

While the Slack channel is essentially public (but very minimal members), I think based on other conversations we've been having about future of editor that folks would expect this to only be visible to their own team members?

Reply:
You're right, thanks for that reminder - since these tabs are within flow settings menu they are already hidden from anyone else (aka `teamViewer`) by default - that was my oversight! 

No need to do anything further here then in my opinion.